1-js/06-advanced-functions/02-rest-parameters-spread-operator/article.md

```diff
@@ -190,56 +189,55 @@ let arr2 = [8, 9, 15];
 let merged = [0, ...arr, 2, ...arr2];
 */!*
 
-alert(merged); // 0,3,5,1,2,8,9,15 (0, then arr, then 2, then arr2)
+alert(merged); // 0,3,5,1,2,8,9,15 (0, depois arr, depois 2, depois arr2)
 ```
 
-In the examples above we used an array to demonstrate the spread operator, but any iterable will do.
+Nos exemplos acima usamos um _array_ para demonstrar o operador _spread_, mas qualquer iterável também funcionaria.
 
-For instance, here we use the spread operator to turn the string into array of characters:
+Por exemplo, aqui usamos o operador _spread_ para transformar uma string em um _array_ de caracteres:
 
 ```js run
-let str = "Hello";
+let str = "Olá";
 
-alert( [...str] ); // H,e,l,l,o
+alert( [...str] ); // O,l,á
 ```
 
-The spread operator internally uses iterators to gather elements, the same way as `for..of` does.
+Internamente, o operador _spread_ usa iteradores _(iterators)_ para reunir elementos, da mesma forma como `for..of` faz.
 
-So, for a string, `for..of` returns characters and `...str` becomes `"H","e","l","l","o"`. The list of characters is passed to array initializer `[...str]`.
+Então, para uma string, `for..of` retorna caracteres, e `...str` se torna `"O","l","á"`. A lista de caracteres é passada para o inicializador do _array_ `[...str]`
 
-For this particular task we could also use `Array.from`, because it converts an iterable (like a string) into an array:
+Para essa tarefa em particular nós também poderíamos usar `Array.from`, porque ele converte um iterável (como uma string) em um _array_.
 
 ```js run
-let str = "Hello";
+let str = "Olá";
 
-// Array.from converts an iterable into an array
-alert( Array.from(str) ); // H,e,l,l,o
+// Array.from converte um interável em um _array_
+alert( Array.from(str) ); // O,l,á
 ```
 
-The result is the same as `[...str]`.
-
-But there's a subtle difference between `Array.from(obj)` and `[...obj]`:
+O resultado é o mesmo que `...[str]`.
 
-- `Array.from` operates on both array-likes and iterables.
-- The spread operator operates only on iterables.
+Mas há uma diferença sútil entre `Array.from(obj)` e `[...obj]`:
 
-So, for the task of turning something into an array, `Array.from` tends to be more universal.
+- `Array.from` opera tanto em listas-genéricas _(array-likes)_ como em iteráveis.
+- O operador spread opera somente em iteráveis.
```

Comment:
Please,

Would you add emphasis to *spread*?
(Only to that, it is already good.)